spoolsense_installer/firmware.py

```
@@ -110,38 +117,65 @@ def verify_flash(port: Optional[str], board_key: str) -> bool:
     flash_match = re.search(r"(\d+)\s*MB", output)
-    if flash_match:
-        flash_bytes = int(flash_match.group(1)) * 1024 * 1024
-        if flash_bytes < min_flash:
-            print(f"\n  ✗ Flash too small!")
-            print(f"    Required: {min_flash // (1024*1024)}MB")
-            print(f"    Detected: {flash_match.group(1)}MB")
-            print(f"    This board cannot run SpoolSense firmware.")
-            sys.exit(1)
-        print(f"  Flash: {flash_match.group(1)}MB ✓")
+    if not flash_match:
+        print(f"\n  ✗ Could not determine flash size from esptool output.")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)